stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -0,0 +1,170 @@
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ template "artifactory.node.name" . }}
+  labels:
+    app: {{ template "artifactory.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: {{ .Values.artifactory.name }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  serviceName: "{{ template "artifactory.node.name" . }}"
+  replicas: {{ .Values.artifactory.node.replicaCount }}
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
```

Comment:
The `app` label should be on all selectors.

Reply:
Will update